Client/Application/SceneDelegate.swift

```diff
@@ -48,7 +48,20 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             sceneCoordinator = SceneCoordinator(scene: scene)
             sceneCoordinator?.start()
 
-            // FXIOS-5827: Handle deeplinks from willConnectTo
+            if let context = connectionOptions.urlContexts.first,
+               let route = routeBuilder.makeRoute(url: context.url) {
+                sceneCoordinator?.handle(route: route)
+            }
+
+            if let activity = connectionOptions.userActivities.first,
+               let route = routeBuilder.makeRoute(userActivity: activity) {
+                sceneCoordinator?.handle(route: route)
+            }
+
+            if let shortcut = connectionOptions.shortcutItem,
+               let route = routeBuilder.makeRoute(shortcutItem: shortcut) {
+                sceneCoordinator?.handle(route: route)
+            }
```

Comment:
Should these be mutually exclusive or is there a scenario where you would want multiple to fire at the same time?

Reply:
I initially had return statements but then I took a look [here](https://github.com/mozilla-mobile/firefox-ios/blob/5beb450745e73a4cc2d70ec77c42db7e2e94560e/Client/Application/SceneDelegate.swift#L213) and it seems to handle all the cases. Also user activity is missing from the current code.

Reply:
I wonder if we could find some doc about this? Could `connectionOptions` contain more than one (URL/user activity, etc) at once? Is there best practice there?